suite/fuzz/drivermc.c

```diff
@@ -90,8 +90,12 @@ int main(int argc, char** argv)
             } else if (strcmp(arch, "CS_ARCH_M680X") == 0 && strcmp(mode, "CS_MODE_M680X_6809") == 0) {
                 Data[0] = 24;
             } else if (strcmp(arch, "CS_ARCH_EVM") == 0 && strcmp(mode, "0") == 0) {
-                Data[0] = 25;
-            } else if (strcmp(arch, "CS_ARCH_BPF") == 0 && strstr(mode, "CS_MODE_BPF_CLASSIC") != NULL) {
+                Data[0] = 25; 
```

Comment:
Thanks @fanfuqiang 

Please rebase on latest `next` branch as there were some changes in the last days.
Then use consistently the latest numbers from fuzz_disasm.c (that would start now at item 44)